src/editors/data/services/lms/api.js

```diff
@@ -0,0 +1,48 @@
+import * as urls from './urls';
```

Comment:
Can we rename this to ...../cms/api.js ? Its a little misleading.

Reply:
sure!  This is copying over the service from gradebook.  That is reasonable